apps/desktop/src/key-management/biometrics/os-biometrics.service.ts

```diff
@@ -1,32 +1,70 @@
 export interface OsBiometricService {
-  osSupportsBiometric(): Promise<boolean>;
+  /**
+   * Authenticates the system user using biometrics.
+   */
+  authenticateBiometric(): Promise<boolean>;
+
+  /**
+   * Checks if biometrics are available on the system.
+   */
+  isBiometricsAvailable(): Promise<boolean>;
+
+  /**
+   * Unlocks the biometrics protected key for a user.
+   * On windows, if masterpassword reprompt is activated, a clientkeyhalf should be passed in
+   */
+  unlockWithBiometrics(
+    userId: string,
+    clientKeyHalfB64: string | undefined,
+  ): Promise<string | null>;
+
+  /**
+   * Enables biometrics unlock for a user. This requires user authentication.
+   * If biometric unlock is already active, resolves to true
+   * If biometric unlock was configured successfully, resolves to true
+   * Else resolves to false
```

Comment:
🎨 Dots `.` at the end of each sentence
```suggestion
   * If biometric unlock is already active, resolves to true.
   * If biometric unlock was configured successfully, resolves to true.
   * Else resolves to false.
```